action.yml

```diff
@@ -27,28 +27,32 @@ runs:
       - name: "Download CI scripts"
         run:  git clone https://github.com/gap-system/pkg-ci-scripts.git
         shell: bash
-      - name: "Install dependencies"
+      - name: "Install dependencies (Linux)"
         run: |
+          if [ ! "${{runner.os}}x" = "Linuxx" ]; then
```

Comment:
The x trick is not necessary, we know this is bash and not a 20 year old broken shell on IRIX or SunOS

Reply:
I used that in other cases, e.g. when using `${{matrix.ABI}}`, to protect against the situation where `matrix.ABI` was not set. Then bash would read `if [  = "32" ]; then`, and complain with `unary operator expected`.

But here `runner.os` will never be empty, and even if it could be empty, then writing
`if [ ! "${{runner.os}}" = "Linux" ]; then` would also protect against that. So yeah, I'll get rid of that.